build.sbt

```diff
@@ -60,25 +60,6 @@ lazy val microservice = Project(appName, file("."))
     Test / unmanagedSourceDirectories ++= Seq(baseDirectory.value / "test", baseDirectory.value / "test-utils"),
```

Comment:
Do we need to state "test" explicitly as a source directory? Elsewhere it's added by default. I realise this is from past work.

Reply:
Nope